.github/workflows/build.yml

```diff
@@ -436,7 +437,7 @@ jobs:
 
             # windows is slow, so dont build unless its a full run
             # - is-full-run: false
-            #   os: windows-2022
+            #   os: windows-2019
```

Comment:
uncomment this block

Reply:
We should leave at least one version building, I'll leave 3.12 on

Reply:
We should leave at least one version building, I'll leave 3.12 on

Reply:
Also the comment isnt true, the windows build turned out to be rather fast so not sure if that changes your thoughts on this

Reply:
yeah they're pretty fast with vcpkg/ccache caching, but they cause the github actions cache to thrash which makes all teh builds longer so we still want to minimize how many we do regularly (not necessary just windows, across all envs)